mail_service/worker_notify_user.py

```diff
@@ -372,12 +372,16 @@ def remove_status_files(self, status_files):
     dfh = config.DynamicFileHandler(
         logger,
         os.path.join(job_info['logs_dir'], config.SERVICE_LOGFILE))
-    logger.info("Received Job:")
-    logger.debug(str(job_info))
-    notify_user = NotifyUser(job_info)
-    build_status = notify_user.notify_user()
-    job.delete()
-    dfh.remove()
+    logger.info("Received Job: {}".format(str(job_info)))
+    try:
+        notify_user = NotifyUser(job_info)
+        build_status = notify_user.notify_user()
+    except Exception as e:
+        logger.critical(
+            "Mail server could not pross the job: {}".format(str(job_info)))
```

Comment:
you mean it could not "process" the job?

Reply:
ah! thanks